e2e-tests/demand-backup-physical-sharded-parallel/run

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -o errexit
+
+test_dir=$(realpath "$(dirname "$0")")
+. "${test_dir}/../functions"
+set_debug
+
+create_infra "${namespace}"
+
+deploy_minio
+apply_s3_storage_secrets
+
+### Case 1: Backup on sharded cluster
+desc 'Testing on sharded cluster'
+
+echo "Creating PSMDB cluster"
+cluster="some-name"
+kubectl_bin apply -f "${conf_dir}/secrets.yml"
+apply_cluster "${test_dir}/conf/${cluster}-sharded.yml"
+kubectl_bin apply -f "${conf_dir}/client_with_tls.yml"
+
+echo "check if all pods started"
+wait_for_running ${cluster}-rs0 3
+wait_for_running ${cluster}-cfg 3
+wait_for_running ${cluster}-mongos 3
+wait_cluster_consistency ${cluster}
+
+lbEndpoint=$(kubectl_bin get svc $cluster-mongos -o=jsonpath='{.status}' \
+	| jq -r 'select(.loadBalancer != null and .loadBalancer.ingress != null and .loadBalancer.ingress != []) | .loadBalancer.ingress[0] | if .ip then .ip else .hostname end')
+if [ -z $lbEndpoint ]; then
+	echo "Mongos service not exported correctly"
+	exit 1
+fi
+
+run_mongos \
+	'db.createUser({user:"myApp",pwd:"myPass",roles:[{db:"myApp",role:"readWrite"}]})' \
+	"userAdmin:userAdmin123456@${cluster}-mongos.${namespace}"
+sleep 1
+run_mongos \
+	'use myApp\n db.test.insert({ x: 100501 })' \
+	"myApp:myPass@${cluster}-mongos.${namespace}"
+sleep 5
+compare_mongos_cmd "find" "myApp:myPass@${cluster}-mongos.${namespace}" "-sharded"
+
+# wait for stable timestamp in wiredtiger
+echo 'Waiting 60 seconds for stable timestamp in wiredtiger'
+sleep 80
+
+echo 'Running backups'
+backup_name_minio="backup-minio-sharded"
+run_backup minio ${backup_name_minio} 'physical'
+if [ -z "$SKIP_BACKUPS_TO_AWS_GCP_AZURE" ]; then
```

Comment:
Is there sense to run this test at all if there is SKIP_BACKUPS_TO_AWS_GCP_AZURE ? We will check minio backup is separate test. Maybe we should move SKIP_BACKUPS_TO_AWS_GCP_AZURE check to the test beginning and skip this test at all ?